index.html

```diff
@@ -2,30 +2,40 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" href="index.css" />
+  <link rel="stylesheet" href="index.css"/>
   <title>Portfolio</title>
 </head>
-
 <body>
-    <nav class="navbar">
+  <header class="header">
+    <div class="container">
       <a class="logo" href="#">My Logo</a>
-      <button type="button" class="hamburger-button">&#8801;</button>
-      <div class="navbar-links">
-        <ul>
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
-         <li><button type="button" class="mail">&#9993;</button></li> 
+      <nav class="navbar">
+        <div class="hamburger-menu">
```

Comment:
- [OPTIONAL] Make sure on the desktop size the hamburger menu button is not visible. Currently, I can see the following below 👇 
![Screenshot (155)](https://user-images.githubusercontent.com/67087664/151942482-c35cee23-e8b6-4afb-9f11-96d699d9c3ad.png)